lib/bolt/cli.rb

```diff
@@ -559,6 +563,50 @@ def initialize_project
       ok ? 0 : 1
     end
 
+    def migrate_project
+      if inventory.version == 2
+        ok = true
+      else
+        inventory_file = config.inventoryfile || config.boltdir.inventory_file
+
+        begin
+          Bolt::Util.file_stat(inventory_file)
```

Comment:
I guess it makes sense to error both case where the default inventory file is not present in a Boltdir and when the specified inventoryfile is not present. For now project migrate only acts on inventory so without an inventoryfile it seems reasonable to error out.